pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md

```diff
@@ -212,6 +214,25 @@ velero
 
 Refer to the [Verify installation][verify-install] topic to ensure successful completion.
 
+### Environments with an HTTP proxy server
+
+The `kommander migrate` command requires a connection from your environment to the Traefik ingress controller in your cluster. If your environment demands the use of an HTTP proxy server, establish an alternative connection to the Traefik ingress controller by running a port-forward into Traefik on a local host.
+
+1.  Ensure you create or modify the `127.0.0.1` record to include the ingress domain name in your hosts file:
+
+    ```bash
+    [root@sv-2000lvp233 kaas-8002-baseline]# cat /etc/hosts
```

Comment:
The # has to be kept here, @mhrabovcin ?

Reply:
```
[root@workstation ~] cat /etc/hosts
```